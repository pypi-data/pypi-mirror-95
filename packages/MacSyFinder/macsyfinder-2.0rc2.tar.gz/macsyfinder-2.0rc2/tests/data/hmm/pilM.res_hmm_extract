PSAE001c01_051130	PSAE001c01	5113	pilM	T4P	2.2e-135	450.2	1.000000
PSAE001c01_044620	PSAE001c01	4462	pilM	T4P	1.6e-15	56.1	0.896450
PRRU001c01_022390	PRRU001c01	2239	pilM	T4P	1.1e-09	36.9	0.952663
