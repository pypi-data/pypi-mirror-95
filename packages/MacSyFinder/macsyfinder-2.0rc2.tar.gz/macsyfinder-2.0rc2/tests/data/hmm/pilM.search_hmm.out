# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.0 (March 2010); http://hmmer.org/
# Copyright (C) 2010 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  ./profiles/pilM.fasta-aln_edit.hmm
# target sequence database:        datatest/prru_psae.001.c01.fasta
# output directed to file:         ./datatest/res_search/pilM.search_hmm.out
# sequence reporting threshold:    E-value <= 1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       pilM.tab.seqid  [M=338]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description
    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------
   1.9e-135  450.4   3.3   2.2e-135  450.2   2.3    1.0  1  PSAE001c01_051130 C GTG TGA 5679649 5680713 Valid pilM 1065 
    4.1e-16   58.0   0.8    1.6e-15   56.1   0.5    1.8  1  PSAE001c01_044620 C ATG TGA 4940535 4941788 Valid ftsA 1254 
    5.6e-10   37.9   2.6    1.1e-09   36.9   1.8    1.4  1  PRRU001c01_022390 D ATG TAA 2818455 2819909 Valid ftsA 1455 
    1.5e-08   33.2   0.0    2.5e-08   32.5   0.0    1.3  1  PSAE001c01_045350 C ATG TAA 5012394 5013431 Valid mreB 1038 
    8.4e-08   30.7   0.0      2e-07   29.5   0.0    1.6  1  PRRU001c01_016340 C ATG TAA 2061250 2062272 Valid mreB 1023 


Domain annotation for each sequence (and alignments):
>> PSAE001c01_051130  C GTG TGA 5679649 5680713 Valid pilM 1065 _PA5044_NP_253731.1_ PA5044 1 5679649 5680713 | type 4 f
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  450.2   2.3  1.3e-138  2.2e-135       1     338 []      13     351 ..      13     351 .. 1.00

  Alignments for each domain:
  == domain 1    score: 450.2 bits;  conditional E-value: 1.3e-138
     pilM.tab.seqid   1 lGidigsssvklvelskqrqgelkLekfaseplpknvvvdgkvvnlaalaqalqqalaeakikkkdvvvavpgsaviikkiqvpdklsekeL 92 
                        lGidi+s+svkl+els+++ g++k+e++a+eplp+n+vv++++v+l++++qal+++l++ak++ k++vvav+gsavi+k+i++++ lse+eL
  PSAE001c01_051130  13 LGIDISSTSVKLLELSRSG-GRYKVEAYAVEPLPPNAVVEKNIVELEGVGQALSRVLVKAKTNLKSAVVAVAGSAVITKTIEMEAGLSEDEL 103
                        7******************.************************************************************************ PP

     pilM.tab.seqid  93 eeaveseaeryipfpleevnlDyqvldlgesdeeekvdVllvAarkeeveeliealeeAGLkpvvldveafalqraeallrevk....erev 180
                        e+++++ea++yip+pleev++D++v+ ++++++ e+vdVll+A+rke+ve + +al+ AGL+++v+dvea+al+r++all+++     ++ +
  PSAE001c01_051130 104 ENQLKIEADQYIPYPLEEVAIDFEVQ-GLSARNPERVDVLLAACRKENVEVREAALALAGLTAKVVDVEAYALERSYALLSSQLgadtDQLT 194
                        **************************.********************************************************99999999* PP

     pilM.tab.seqid 181 vvlvdigaasteiliiqngktlykrelsisgkqlteaiqrafsldlekaeelkrkgglPteyeeellrpvleeltqelrrslqfylatseea 272
                        v++vdiga++t++++++ng+t+y+re+ ++g+qlte+iqr++ l++e+a  +k++gglP++y++e+lrp++++++q+++rslqf++a+ +++
  PSAE001c01_051130 195 VAVVDIGATMTTLSVLHNGRTIYTREQLFGGRQLTEEIQRRYGLSVEEAGLAKKQGGLPDDYDSEVLRPFKDAVVQQVSRSLQFFFAAGQFN 286
                        ******************************************************************************************** PP

     pilM.tab.seqid 273 evekilLaGggaklqglaelitqrlgvpvelvnPfaklaldnkkvdaeqlqedaPalavalGLalR 338
                        +v++i+LaGg+a++q+l++li+q++g+p+ ++nPfa++al n kv+a +l++daPal++a+GLalR
  PSAE001c01_051130 287 DVDYIVLAGGTASIQDLDRLIQQKIGTPTLVANPFADMAL-NGKVNAGALASDAPALMIACGLALR 351
                        ***************************************8.************************9 PP

>> PSAE001c01_044620  C ATG TGA 4940535 4941788 Valid ftsA 1254 _PA4408_NP_253098.1_ PA4408 1 4940535 4941788 | cell div
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   56.1   0.5   9.8e-19   1.6e-15       1     303 [.      11     351 ..      11     361 .. 0.75

  Alignments for each domain:
  == domain 1    score: 56.1 bits;  conditional E-value: 9.8e-19
     pilM.tab.seqid   1 lGidigsssvklvelskqrqgelkLekfaseplpknvvvdgkvvnlaalaqalqqalaea....kikkkdvvvavpgsavii....kkiqvp 84 
                        +G+dig s  k+v l ++   + +Le +++   p   +  g vvn++  +q +q+a+ ea    + + +++ v+++g+   +     ++ + 
  PSAE001c01_044620  11 VGLDIGTS--KVVALVGEVTADGQLEVVGIGTHPSRGLKKGVVVNIESTVQSIQRAIDEAqqmaGCRIHSAFVGIAGNHIRSlnshGIVAIR 100
                        8*****98..7899999999999**********************************99632226899********9875431111223333 PP

     pilM.tab.seqid  85 d.klsekeLe......eaveseaer.yipfpleevnlDyqvl...dlgesdeeekvdVllvAarkeeveeliealeeAGLkpvvldveafal 165
                        d +++  ++e      +av + a++ ++    ++ ++D q     +lg s    + +V +v +  ++ +++ + ++  GL+++ +  e+  l
  PSAE001c01_044620 101 DrEVNPADIErvldaaQAVAIPADQrVLHTLAQDYVIDNQEGvrePLGMSGVRLEAKVHVVTCAVNASQNIEKCVRRCGLEVDDIILEQ--L 190
                        31223333330000114455555431333334556666553211223333333344566677777788899999******988666665..5 PP

     pilM.tab.seqid 166 qraeallrevkerevvvlvdigaasteiliiqngktlykrelsisgkqlteaiqrafsldlekaeelkrkgg..............lP.... 239
                          a+++l+e  ++  v lvdig+ +t i+i+ +g++  + +++i+g+q+t+ i+ a+  +++ aee+k ++               +P    
  PSAE001c01_044620 191 ASAYSVLTEDEKELGVCLVDIGGGTTDIAIFTEGAIRHTAVIPIAGDQVTNDIAMALRTPTQYAEEIKIRYAcalaklagagetikVPsvgd 282
                        68999999999999**********************************************************88888888888877335555 PP

     pilM.tab.seqid 240 ...teyeeellrpvleeltqelrrslqfylats..eeaevekilLaGggaklqglaelitqrlgvpvel 303
                            e   + l +v e   +el   +q  l  s  e+  ++ i+L+Gg++k++g  el ++   +pv l
  PSAE001c01_044620 283 rppRELSRQALAEVVEPRYDELFTLVQAELRRSgyEDLIPAGIVLTGGTSKMEGAVELAEEIFHMPVRL 351
                        5558888888888888888888888887777766699999**************999988888888766 PP

>> PRRU001c01_022390  D ATG TAA 2818455 2819909 Valid ftsA 1455 _PRU_2335_YP_003575591.1_ PRU_2335 1 2818455 2819909 | c
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   36.9   1.8   6.7e-13   1.1e-09       2     323 ..       9     360 ..       8     373 .. 0.73

  Alignments for each domain:
  == domain 1    score: 36.9 bits;  conditional E-value: 6.7e-13
     pilM.tab.seqid   2 GidigsssvklvelskqrqgelkLekfaseplpknvvvdgkvvnlaalaqalqqalae....akikkkdvvvavpgsaviikkiqvpdklse 89 
                         i++gss+v  +  +k+  g++++ +++ ++   + +  g v n++  aq l  ++ +    +k++ ++v v+v g++  + k  v++ l+ 
  PRRU001c01_022390   9 AIELGSSKVTGIAGQKNLDGSINVLAVV-KENSSSFIRKGVVYNIDKTAQCLTSIIKKlenqLKTQITQVYVGVGGQSIRSVKNVVTKDLPG 99 
                        5889999888888888888877776654.4556789999******99998876666552222799999******998765444444445555 PP

     pilM.tab.seqid  90 keL..eeav..eseaeryipfpleev........nlDyqvl.dlgesd.eeekvdVllvAarkeeveeliealeeAGLkpvvldveafalqr 167
                        + +  +++v   ++a+r + ++ +e+        ++ +q + d ++ + ++ + + l +  rk    +l + +e+AG+++  +   + al  
  PRRU001c01_022390 100 ETIitQDMVieLMDANRNMTYQDQEIldaavqeyKVGTQFQlDPVGIQaTHLEGHFLNILERKAFYRNLNKCFETAGINVAEMYLAPLALA- 190
                        54423444544467777777766554111111114555544433333304445577889999999*************9999999888875. PP

     pilM.tab.seqid 168 aeallrevkerevvvlvdigaasteiliiqngktlykrelsisgkqlteaiqrafsldlekaeelkrkgg....lPteyeeell........ 247
                          ++l+ev +r   +lvd+g ++t++++++ +      +++++ +++t+ i+ +++++++ ae++k k+g      +e+++el         
  PRRU001c01_022390 191 -DSVLTEVEKRSGCALVDLGSDTTTVSVYSKNILRHLAVIPLGSNNITKDIA-TLQMEESDAEKMKLKYGsaytDNNEIDNELKysidpdrq 280
                        .678999999*9***************************************9.79***********99995544337777777788899999 PP

     pilM.tab.seqid 248 ...rpvleeltqelrrslqfylats....eeaevekilLaGggaklqglaelitqrlgvpvelvnPfaklaldnkkvdaeqlq 323
                           r++ e +   l+  +++   +      ++ +  i+L+Ggg++++++++++ +++ v+   +  f +  +   ++++e ++
  PRRU001c01_022390 281 vesRKFIEIVEGRLEEIIENVWYQIpseyYDKLLGGIILTGGGSNMKNIEKAFFNHTHVDKIRIAKFVSQTV---QTNNEDIK 360
                        99999**************865555223334456789**********************9888877766665...44444444 PP

>> PSAE001c01_045350  C ATG TAA 5012394 5013431 Valid mreB 1038 _PA4481_NP_253171.1_ PA4481 1 5012394 5013431 | rod shap
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   32.5   0.0   1.5e-11   2.5e-08     177     305 ..     155     317 ..     134     325 .. 0.75

  Alignments for each domain:
  == domain 1    score: 32.5 bits;  conditional E-value: 1.5e-11
     pilM.tab.seqid 177 erevvvlvdigaasteiliiqngktlykrelsisgkqlteai....qrafs..ldlekaeelkrkgg..lP..................... 239
                        e+   ++vdig+ +tei++i+ +  +y   + ++g+ + eai     r++   + ++ ae +k++ g   P                     
  PSAE001c01_045350 155 EARGSMVVDIGGGTTEIALISLNGVVYAESVRVGGDRFDEAIvtyvRRNYGslIGESTAERIKQEIGtaFPggdvrevdvrgrnlaegvprs 246
                        5557789*********************************9844445788554678889999998887777777766666665555555566 PP

     pilM.tab.seqid 240 .....teyeeell...rpvleeltqelrrslqfylats.eeaevekilLaGggaklqglaelitqrlgvpvelvn 305
                             +e+    l   +++l +++q ++ +l+       ++      +L+Ggga l++l++l++q++g+pv +++
  PSAE001c01_045350 247 ftlnsNEV----LealQESLATIVQAVKSALEQSPPELaSDIAERGLVLTGGGALLRDLDKLLAQETGLPVIVAE 317
                        64432222....244488888999999999988776662444455689**********************98874 PP

>> PRRU001c01_016340  C ATG TAA 2061250 2062272 Valid mreB 1023 _PRU_1708_YP_003575003.1_ PRU_1708 1 2061250 2062272 | r
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !   29.5   0.0   1.2e-10     2e-07     177     311 ..     150     322 ..     134     328 .. 0.71

  Alignments for each domain:
  == domain 1    score: 29.5 bits;  conditional E-value: 1.2e-10
     pilM.tab.seqid 177 erevvvlvdigaasteiliiqngktlykrelsisgkqlteaiq....rafsldlek..aeelkrkgg.........lP.............. 239
                        + e  ++vdig+ stei++i+ g ++ +  +  +g++lt  iq    r+ ++++++  ae +k + g          P              
  PRRU001c01_016340 150 APEGNMIVDIGGGSTEIAVISLGGIVSNNSIRTAGDDLTADIQeymyRQHNVKVSErmAERIKINVGsaltdlgdeAPedfivhgpnrital 241
                        5566789********************************99986666777777776115555555557777744442244444444444444 PP

     pilM.tab.seqid 240 ........teyeeellrpvleeltqelrrslqfylats.eeaevekilLaGggaklqglaelitqrlgvpvel.vnPfakla 311
                                +e+  + l +   ++ + +  +l++      ++   + i+L+Ggga l+gl++ +t+++ +p+++  +P++++a
  PRRU001c01_016340 242 pmevpvcyQEIA-HCLDKTVAKIENAVLSALENTPPELyADIVKNGIYLSGGGALLRGLDRRLTDKMHIPFHVpEDPLHSVA 322
                        444444444444.3446666677777777888777776344444569***********************986246777665 PP



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (338 nodes)
Target sequences:                       8332  (2926593 residues)
Passed MSV filter:                       283  (0.0339654); expected 166.6 (0.02)
Passed bias filter:                      195  (0.0234037); expected 166.6 (0.02)
Passed Vit filter:                        21  (0.0025204); expected 8.3 (0.001)
Passed Fwd filter:                         5  (0.000600096); expected 0.1 (1e-05)
Initial search space (Z):               8332  [actual number of targets]
Domain search space  (domZ):               5  [number of targets reported over threshold]
# CPU time: 0.28u 0.01s 00:00:00.29 Elapsed: 00:00:00.11
# Mc/sec: 8992.62
//
